ast/src/main/scala/jawn/ast/JawnFacade.scala

```diff
@@ -5,9 +5,9 @@ import scala.collection.mutable
 
 object JawnFacade extends Facade[JValue] {
 
-  final val jnull = JNull
-  final val jfalse = JFalse
-  final val jtrue = JTrue
+  final def jnull(): JValue = JNull
```

Comment:
Wouldn't it make sense to remove the empty parens in the facade trait, instead of adding them in the implementations?

Reply:
I left them mostly because I [want to remove these methods entirely](https://gitter.im/non/jawn?at=5e148fb7559755189037c739), and was trying to minimize changes.

Also using `val` is generally going to be an anti-optimization for these, since they'll usually be singletons and a simple method returning them is better than an extra field.

Reply:
Do you want to remove them pre-1.0.0?

In any case, the changes you made are conservative wrt the existing trait, so I don't feel strongly _against_ the parens.

Reply:
Yes, the idea would be to flatten e.g. `RawFacade` and `Facade` into just `Facade`. I'll open a PR in the morning for discussion.